css/cssom-view/resources/matchMedia.js

```diff
@@ -0,0 +1,32 @@
+const IFRAME_BASE_WIDTH = "200";
```

Comment:
With minor refactoring, a few existing tests will benefit from these helpers in separate file:
* [matchMedia.xht](https://github.com/web-platform-tests/wpt/blob/master/css/cssom-view/matchMedia.xht)
* [matchMediaAddListener.html](https://github.com/web-platform-tests/wpt/blob/master/css/cssom-view/matchMediaAddListener.html)

As well as some new tests I am working on (`onchange`, `extends EventTarget`).